.github/workflows/scip-typescript.yml

```diff
@@ -13,9 +13,10 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - name: Install npm dependencies
-        run: yarn
-      - run: yarn generate
-      - run: yarn dlx -p @sourcegraph/scip-typescript scip-typescript index --yarn-berry-workspaces
+        run: pnpm install
+      - run: pnpm generate
+      # TODO: dlx?
+      - run: pnpm dlx -p @sourcegraph/scip-typescript scip-typescript index --yarn-berry-workspaces
```

Comment:
`yarn berry`

Reply:
I think I wasn't finished this line

Reply:
See https://yarnpkg.com/cli/dlx. Pnpm supports this too.

Reply:
But has different syntax, so I'll update this.